js/read_dir_test.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+import { test, testPerm, assert, assertEqual } from "./test_util.ts";
+import * as deno from "deno";
+
+testPerm({ write: true }, function readDirSyncSuccess() {
+  const files = deno.readDirSync("tests/");
+  
+  for (const file of files) {
+    assert(file.isFile() || file.isDirectory());
+
+    if (file.isDirectory()) {
+      assertEqual(file.name, 'subdir');
+    }
```

Comment:
This will break if we add another subdirectory. Please invert the test:
```js
if (file.name == 'subdir') assert(file.isDirectory())
```

Reply:
Fixed.